azurerm/internal/services/healthcare/healthcare_service_resource.go

```diff
@@ -2,10 +2,12 @@ package healthcare
 
 import (
 	"fmt"
+	keyVaultParse "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/keyvault/parse"
+	keyVaultValidate "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/keyvault/validate"
```

Comment:
Can we move these imports into the next section, with the other imports from the same module?

Reply:
Fixed